### PR TITLE
bugfix: deduplicate errors before logging them more than once in the event

### PR DESCRIPTION
**Pull request by mitsuhiko**
This fixes the issue where we end up with lots of duplicated error data in storage.

### CODE REVIEW

File: src/sentry/stacktraces.py [hunk 1]
```diff
@@ -380,6 +380,16 @@ def get_stacktrace_processing_task(infos, processors):
     )
 
 
+def dedup_errors(errors):
+    # This operation scales bad but we do not expect that many items to
```
Comment (mattrobenolt) [inline src/sentry/stacktraces.py]: I mean, this is fine, but why not just do like, `return list(set(errors))` ? Or are you intentionally trying to preserve the order?

**Review by mattrobenolt — APPROVED** [review]

File: src/sentry/stacktraces.py (hunk 1 above)
Comment (mitsuhiko) [reply src/sentry/stacktraces.py]: That and more importantly: errors are not hashable.

Comment (mattrobenolt) [reply src/sentry/stacktraces.py]: 🤔

Comment (mitsuhiko) [reply src/sentry/stacktraces.py]:
```pycon
>>> set([{'foo': 'bar'}])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'dict'
```

**Comment by mattrobenolt** [comment]
Hi.

**Pull request merged**